include/Gaffer/Context.h

```diff
@@ -254,6 +254,15 @@ class GAFFER_API Context : public IECore::RefCounted
 
 				const Context *context() const { return m_context.get(); }
 
+				// Forward compatibility with Gaffer 0.60
+				template<typename T, typename = typename std::enable_if< !std::is_base_of<IECore::Data,T>::value >::type >
+				void set( const IECore::InternedString &name, const T *value );
+				template<typename T, typename = typename std::enable_if< !std::is_pointer<T>::value >::type >
+				void setAllocated( const IECore::InternedString &name, const T &value );
+				void setAllocated( const IECore::InternedString &name, const IECore::Data *value );
+				void setFramesPerSecond( const float *framesPerSecond );
+
+
```

Comment:
Just one blank line please - the extra spacing doesn't help readability and means we get less code on screen.